Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM quay.io/ukhomeofficedigital/alpine:v3.13
+
+ENV USER user-hocs-bulk-case-action
+ENV USER_ID 1000
+ENV GROUP group-hocs-bulk-case-action
+ENV NAME hocs-bulk-case-action
+ENV JAR_PATH build/libs
+
+ENV workflow-address http://host.docker.internal:8091
```

Comment:
This can just be moved into the application properties so can be overriden by K8 env variable pass through.

Reply:
This and the one below are in the application.properties

Reply:
But if it's here we cannot overwrite it? When the image starts up regardless of what the param we passthrough is, this layer then gets hit?

Reply:
Ah right yeah gottcha

Reply:
Removed now from Dockerfile so can be set in K8